src/config.json

```diff
@@ -19,7 +19,9 @@
         "extension_registry": "https://s3.amazonaws.com/extend.brackets/registry.json",
         "extension_url": "https://s3.amazonaws.com/extend.brackets/{0}/{0}-{1}.zip",
         "linting.enabled_by_default": true,
-        "build_timestamp": ""
+        "build_timestamp": "",
+        "healthDataServerURL" : "http://10.192.16.236:8080",
```

Comment:
@peterflynn At the time of raising the PR, integration of server was not done so I used local machine's ip as placeholder. We are working on switching to https and domain name and will update on client as soon as we are done with it